docs/guides/interactive-designer.md

```diff
@@ -0,0 +1,29 @@
+# Interactive designer
```

Comment:
This is good, maybe add some callout that if this doesn't work for you, contact us -- and link to support.